src/main/java/org/example/Repository.java

```diff
@@ -62,6 +65,30 @@ public boolean passwordValidator(String password) {
         Pattern pattern = Pattern.compile("^(?=.*[0-9].*[0-9])(?=.*[A-Z])(?=.*[!@£$%^&*()_+=,./?><:|]).{8,}$");
         return pattern.matcher(password).matches();
     }
+
+    /////////////////////////////////////////////////////  POLICY SERVICE
+
+    public int getCustomerId(String username) throws SQLException, IncorrectUsernamePasswordException {
+        ResultSet resultSet = connect.createStatement().
+                executeQuery(String.format(" SELECT UserId FROM usersregistration WHERE user_username = '%s'", username));
+        if (resultSet.next()) {
+            return resultSet.getInt("UserId");
+        }
+        throw new IncorrectUsernamePasswordException("Incorrect username or password");
```

Comment:
I don't think we need this. We should have the customerId already present on the customer when we need to call getListPolicyTypes

Reply:
Deleted!